packages/react-widgets/src/models/utils/checkParams.js

```diff
@@ -0,0 +1,7 @@
+export default function checkParams ({ joinOperation, operationColumn }) {
```

Comment:
This name is too generic if it's only meant to test this particular kind of relation between params.

Reply:
Removed, it makes more sense to use the custom prop type validator instead of checking in the model.

Reply:
@Clebal I don't know, prop type is only a warning for the dev but doesn't block in case of error. What happens with the components in case of invalid props? I guess if they don't crash everything that's ok :smile:

Reply:
Idk if the app should break. @borja-munoz what do you think? Should we break the app if joinOperation isn't defined? Or should we just show a warning in console?

Reply:
I think it should break. It should behave in the same way that it behaves if we miss a required prop (i.e. `dataSource`).

Reply:
Fixed.